aws/sqs_event_manager/README.rst

```diff
@@ -1,3 +1,89 @@
 AWS SQS interface
 
 React on SQS events connected as lambdas. 
+
+EVENTS
+------
+
+The SNS Topics have the following event format:
+
+.. code::
+
+   {
+      "Records": [
+          {
+              "messageId": "uuid",
+              "receiptHandle": "string",
+              "body": {
+                  "Type" : "Notification",
+                  "MessageId" : "uuid",
+                  "TopicArn" : "arn:aws:sns:us-east-1:XXX:aws-mp-entitlement-notification-XXX",
+                  "Message" : {
+                      "action": "entitlement-updated",
+                      "customer-identifier": "string",
+                      "product-code": "string"
+                  },
+                  "Timestamp" : "2025-01-15 16:31:50",
+                  "SignatureVersion" : "1",
+                  "Signature" : "abc123",
+                  "SigningCertURL" : "string",
+                  "UnsubscribeURL" : "string"
+              },
+              "attributes": {
+                  "ApproximateReceiveCount": "1",
+                  "SentTimestamp": "1545082649183",
+                  "SenderId": "string",
+                  "ApproximateFirstReceiveTimestamp": "1545082649185"
+              },
+              "messageAttributes": {},
+              "md5OfBody": "string",
+              "eventSource": "aws:sqs",
+              "eventSourceARN": "arn:aws:sqs:us-east-1:111122223333:my-queue",
+              "awsRegion": "us-east-1"
+          }
+      ]
+  }
+
+REQUEST
+--------
+
+.. code::
+
+      {
+          "marketplaceIdentifier": "AWS",
+          "customerIdentifier": "CustomerIdentifier"
+          "entitlements": [
+              {
+                  "customerIdentifier": "string",
+                  "productCode": "string",
+                  "expirationDate": 123123111231,
+                  "dimension": "string",
+                  "value": {
+                      "booleanValue": true|false,
+                      "doubleValue": 1,
+                      "integerValue": 2,
+                      "stringValue": "string"
+                  }
+              }
+          ]
+    }
+
+ERROR RESPONSE
+--------------
+
+.. code::
+
+    {
+        "isBase64Encoded": false,
+        "statusCode": HTTP_STATUS_CODE,
+        "body": {
+            "errors": {
+                "Registration": "MESSAGE",
+                "Exception": "AWS. or App. error code"
+            }
+        }
+    }
+
+* 500: Internal Server Error
+* 503: Service unavailable, any non AWS ClientException
```

Comment:
Great thanks for the update. The 500 and 503 are covered by exception names as requested from the SCC team. See #13 for details. If you can update that for consistency would be great and then this can be merged. Thanks much

Reply:
Any error handling for SQS code would be sent back to the sns event originator not SCC. However, the code should mostly log error cases and skip the message in the queue and try later.

Reply:
I know and that's fine. What I meant is the description for 500 and 503 which you copied from my README but it will be changed in #13 to this

```
500: App.Error.InternalServiceErrorException
503: App.Error.ServiceUnavailableException
```

Alex requested exception names for this cases and I implemented them in #13. It would be nice if we can stay consistent regarding 500 and 503. Alternatively you could also consider to not mention the 500/503 at all like you already said errors are sent back to the creator of the event and are not handled by SCC. Nevertheless I like consistency :) 

Thoughts ?

Reply:
perfect, thanks. I'll merge a good number of the open PRs after the meeting.